README.md

```diff
@@ -10,27 +10,11 @@ Internally, the Conjur Buildpack uses [Summon](https://cyberark.github.io/summon
 
 ## Requirements
 
-+ The Conjur Buildpack **requires** the `meta-buildpack` to work.
++ The Conjur Buildpack **requires** the `meta-buildpack` to work as a buildpack otherwise you will need to manually load the buildpack scripts.
```

Comment:
How about something like

"The Conjur Buildpack requires the meta-buildpack. If you are unable to use the meta-buildpack (because you are using a custom buildpack, for example), see the *instructions below* (link) for manually loading the buildpack scripts."